src/plugins/core/sheet.ts

```diff
@@ -639,10 +639,16 @@ export class SheetPlugin extends CorePlugin<SheetState> implements SheetState {
   }
 
   private checkSheetName(cmd: RenameSheetCommand | CreateSheetCommand): CommandResult {
+    const originalSheetName = this.getters.tryGetSheetName(cmd.sheetId);
+    if (originalSheetName !== undefined && cmd.name === originalSheetName) {
```

Comment:
same as this , right ?
```suggestion
    if (cmd.name === originalSheetName) {
```

Reply:
@LucasLefevre Since `checkSheetName` is used elsewhere, `originalSheetName` is possibly `undefined` if the sheet doesn't exist.